lib/commands/create.js

```diff
@@ -98,3 +110,36 @@ module.exports = function (program) {
 function replaceInConfig(regex, replacement) {
   util.replaceInFiles([ './app.js', './package.json' ], regex, replacement);
 }
+
+async function spanWithSpinner () {
+  const spinner = ora({
+    text: 'Installing packages...',
+    // stream: process.stdout,
+    interval: 100,
+    isEnabled: true
+  });
+
+  return new Promise((resolve, reject) => {
+    try {
+      const installer = spawn('npm install', {
+        shell: true
+      });
+
+      spinner.start();
+
+      installer.on('exit', (code) => {
+        spinner.stop();
+
+        if (code !== null && code !== 0) {
+          reject(new Error(`Install spawn process error with code ${code}`));
```

Comment:
I think it's a good idea to "return reject(...)", "return resolve(...)", etc. You successfully avoided any inadvertent fallthroughs here but it's a good pattern to make sure that remains the case.